### PR TITLE
IGNITE-20954 Prohibit the same group for event and metadata consumers

### PR DESCRIPTION
**Pull request by NSAmelchev**

### CODE REVIEW

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/AbstractKafkaToIgniteCdcStreamer.java [hunk 1]
```diff
@@ -107,9 +107,14 @@ public AbstractKafkaToIgniteCdcStreamer(Properties kafkaProps, KafkaToIgniteCdcS
         appliers = new ArrayList<>(streamerCfg.getThreadCount());
         runners = new ArrayList<>(streamerCfg.getThreadCount());
 
-        if (!kafkaProps.containsKey(ConsumerConfig.GROUP_ID_CONFIG))
+        String grp = kafkaProps.getProperty(ConsumerConfig.GROUP_ID_CONFIG);
+
+        if (grp == null)
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/AbstractKafkaToIgniteCdcStreamer.java:112]: Can we make this check with `A.ensure` like other parameters?

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/AbstractKafkaToIgniteCdcStreamer.java [hunk 2]
```diff
@@ -107,9 +107,14 @@ public AbstractKafkaToIgniteCdcStreamer(Properties kafkaProps, KafkaToIgniteCdcS
...
             throw new IllegalArgumentException("Kafka properties don't contains " + ConsumerConfig.GROUP_ID_CONFIG);
 
+        if (grp.equals(streamerCfg.getMetadataConsumerGroup()))
```
Comment (nizhikov) [inline modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/AbstractKafkaToIgniteCdcStreamer.java:115]: Can we make this check with `A.ensure` like other parameters?

**Review by nizhikov — APPROVED** [review]

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/AbstractKafkaToIgniteCdcStreamer.java (hunk 1 above)
Comment (NSAmelchev) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/AbstractKafkaToIgniteCdcStreamer.java]: Fixed

File: modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/AbstractKafkaToIgniteCdcStreamer.java (hunk 2 above)
Comment (NSAmelchev) [reply modules/cdc-ext/src/main/java/org/apache/ignite/cdc/kafka/AbstractKafkaToIgniteCdcStreamer.java]: Fixed

**Pull request merged**